client/src/js/services/DebtorGroupService.js

```diff
@@ -35,6 +35,10 @@ function DebtorGroupService($http, util) {
   * @description This function is responsible for create new debtor group
   */
   function create(debtorGroup) {
+
+    // augment object with session values 
+    debtorGroup.enterprise_id = SessionService.enterprise.id;
```

Comment:
I agree that setting this on the server would be a good idea. This would clean up the client code. 

We should make sure that having seen this pattern implemented it is still clear to developers where and why the enterprise information is being set - we should make sure it's not just 'magic'. 

If it seems like a good pattern we can create an issue to track standardising this across all 2.x services/ APIs.